doc/user-guide/user-guide.rst

```diff
@@ -0,0 +1,598 @@
+        dec=Angle(-80., unit=u.deg),
+        rot_sky=0.,
+        target_name="Test target"
+    )
+
+The :py:meth:`slew_icrs <lsst.ts.observatory.control.BaseTCS.slew_icrs>` also implements a couple different rotator positioning strategies.
+The most common strategy is to use ``rot_sky``, also known as position angle (PA), the angle between north direction and the bore-sight y-axis, measured in the eastward direction.
+By default ``rot_sky=0.`` and it can be changed by passing in the desired value;
+
+.. code:: python
+
+    await tcs.slew_icrs(
+              ra=20.0, dec=-80.0, rot_sky=90., target_name="Test target"
+          )
+
+Users also have the option to select a physical angle for the rotator.
+For instance, if you are trying to keep the Rotator close to a particular physical range (due to some hardware limitation or observational strategy), use ``rot_phys_sky`` instead;
+
+.. code:: python
+
+    await tcs.slew_icrs(
+              ra=20.0, dec=-80.0, rot_phys_sky=20., target_name="Test target"
+          )
+
+This will cause the rotator to be positioned close to the physical (e.g. encoder) angle of ``20.`` degrees.
+Not that this angle is defined at the start of the slew, and the telescope will resume tracking normally, so the rotator will be moving to de-rotate the field.
+
+If instead, you need the rotator to remain fixed at a set position but the telescope must track (e.g. for filter changes on the main telescope), use the ``rot_phys`` option.
+
+.. code:: python
+
+    # WARNING: The telescope will track the alt/az axis but the rotator will
+    # be kept fixed in physical position 0. degrees.
+    await tcs.slew_icrs(
+              ra=20.0, dec=-80.0, rot_phys=0., target_name="Test target"
+          )
+
+When conducting spectroscopy (e.g. with the Auxiliary Telescope) it is useful to be able to position the field in terms of the parallactic angle.
+For that, one can use the ``rot_par`` parameter;
+
+.. code:: python
+
+    await tcs.slew_icrs(
+              ra=20.0, dec=-80.0, rot_par=0., target_name="Test target"
+          )
+
+Although ``rot_par=0.`` is the most commonly used value, the user is free to select any angle.
+
+In case the user selects an angle that would cause the rotator to slew to a position outside the valid range, the task will fail and raise an exception.
+
+  >>> await tcs.slew_icrs(...)
+  ---------------------------------------------------------------------------
+  AckError                                  Traceback (most recent call last)
+  <ipython-input-25-be270f3a125b> in async-def-wrapper()
+  .
+  .
+  .
+  AckError: msg='Command failed', ackcmd=(ackcmd private_seqNum=1597989109,
+  ack=<SalRetCode.CMD_FAILED: -302>, error=6611,
+  result='Rejected : rotator position angle out of range')
+
+The error message will show the exception traceback, which can be somewhat intimidating.
+However, the important bit of information can be found in the last couple lines of the output (as shown above).
```

Comment:
Is there something that indicates which CSC the error comes from? It'd be nice to show that as well since the exceptions aren't often thrown from the TCS class..

Reply:
uhm... that is a good point... unfortunately, I don't think that information is readily available... there are a couple ways we could go about it either salobj can be more explicit about who rejects the command or the module could be... not sure what would be best.... I think salobj philosophy is that the user knows to which CSC it sent the command to, but this is not true in this case...